var/spack/repos/builtin/packages/care/package.py

```diff
@@ -21,6 +21,18 @@ class Care(CMakePackage, CudaPackage, ROCmPackage):
 
     version("develop", branch="develop", submodules="True")
     version("master", branch="main", submodules="True")
+    version(
+        "0.13.3",
+        tag="v0.13.3",
+        commit="93853696b452647278eae9311b835ad206236522",
+        submodules="True",
```

Comment:
I don't think I understood this before, but I don't think we need the submodules.

Reply:
OK, will have to test that change in our CI.
Would you know whether they were needed at some point ?

Reply:
I'm not sure they were ever needed.